### ISSUE TITLE
make on go v1.14: flag provided but not defined: -mod

### ISSUE THREAD

**Issue by tstromberg** [GoogleContainerTools/skaffold#3699]
If you run `make` with `go go1.14rc1`, it errors with:

```
hack/generate-statik.sh
Installing go-licenses
/var/folders/n1/qxvd9kc11w15mc4qv6g43x80000kt3/T/tmp.wlxi6kFN ~/src/skaffold
go: creating new go.mod: module tmp
flag provided but not defined: -mod
usage: go get [-d] [-t] [-u] [-v] [-insecure] [build flags] [packages]
Run 'go help get' for details.
make: *** [generate-statik] Error 2
```

This flag was removed in https://go-review.googlesource.com/c/go/+/198438/

**Comment by dgageot** [comment]
Thanks @tstromberg for reporting that issue. What about removing that flag once go 1.14 is released and we switch to using it in our development guide and CI/CD?

**Comment by dgageot** [comment]
This was fixed with #3775
